src/jloptions.c

```diff
@@ -18,6 +19,15 @@ char *shlib_ext = ".dylib";
 char *shlib_ext = ".so";
 #endif
 
+/* This simple hand-crafted tolower exists to avoid locale-dependent effects in
+ * behaviors (and utf8proc_tolower wasn't linking properly on all platforms) */
+static char ascii_tolower(char c)
+{
+    if ('A' <= c && c <= 'Z')
+        return c - 'A' + 'a';
```

Comment:
So, let me recap: this PR changed things to use `tolower`, which has issues, so it was suggested to use `utf8proc_tolower` instead which then was replaced by this hand-written function...

How about my original suggestion to just stick with what the code did before this PR, and simply have two cases in the `switch` for each letter? That seems by far simplest and is guaranteed to have no UTF-8 related issues or anything else and requires no additional explanation?

Reply:
Haha, yeah. But this works too, and there are a couple extra cases now, including one that is not in a switch statement, so it is not entirely obvious to me that it is significantly better one way or the other; just a style preference.